hexl/experimental/seal/ckks-multiply.cpp

```diff
@@ -35,6 +35,7 @@ void CkksMultiply(uint64_t* result, const uint64_t* operand1,
   size_t num_tiles = n / tile_size;
 
   AlignedVector64<uint64_t> temp(tile_size, 0);
+  AlignedVector64<uint64_t> temp2(tile_size, 0);
```

Comment:
Only one temp variable is enough to use it in the first multiplication of the second polynomial.

Reply:
done.